app/static/i18n/fr.json

```diff
@@ -0,0 +1,147 @@
+        "CLEAR_HISTORY": "Effacer l'historique",
+        "CLEAR_HISTORY_MSG": "l'entrée dans l'historique de la console a été effacée.",
+        "CLEAR_HISTORY_MSG_PLURAL": "les entrées dans l'historique de la console ont été effacées.",
+        "EXCUTE_QUERY": "Executer la Requête",
+        "SHOW_ERROR_TRACE": "Voir la trace de l'erreur",
+        "SHOW_ERROR_TRACE_HINT": "Selectionnez `Voir la trace de l'erreur` dans les Options de la console pour afficher la trace de l'erreur.",
+        "ERROR": "Erreur!",
+        "RESULT_FROM_QUERY": "Résultat de la requête",
+        "ERROR_TRACE": "TRACE DE L'ERREUR",
+        "FORMAT_RESULTS": "Formatter les résultats",
+        "UNSAFE_INTEGER": "Ce nombre excède l'entier maximum sûr de JavaScript. Pour être certain d'obtenir le bon nomber, coercez-le dans votre requête SQL en utilisant 'field::string'."
+    },
+    "TABLE": {
+        "TITLE": "Tables",
+        "NAME": "Nom",
+        "HEALTH": "Santé",
+        "CONFIG_REPLICAS": "Réplications Configurées",
+        "CONFIG_SHARDS": "Silos Configurés",
+        "STARTED_SHARDS": "Silos Démarrés",
+        "MISSING_SHARDS": "Silos Manquants",
+        "UNDERREPL_SHARDS": "Silos Sous-Répliqués",
+        "TOTAL": "Total",
+        "UNAVAILABLE": "Non-Disponible",
+        "UNDERREPL": "Sous-Repliqués",
+        "SIZE": "Taille",
+        "RECOVERY": "Recovery",
+        "BLOBS": "Blobs",
+        "RECORDS": "Entrées",
+        "PARTITION_COLUMNS": "Colonne de Partition",
+        "PARTITIONS": "Partitions",
+        "ROW": {
+            "IDENT": "Identation",
+            "PARTITION_VALUES": "Valeur de Partition",
+            "CONF_REPLICAS": "Réplications Configurées",
+            "CONF_SHARDS": "Silos Configurés",
+            "STARTED_SHARDS": "Silos Démarrés",
+            "MISSING_SHARDS": "Silos Manquants",
+            "UNDERREPL_SHARDS": "Silos Sous-Répliqués",
+            "TOTAL_RECORDS": "Nombre total d'Entrées",
+            "UNAVAIL_RECORDS": "Entrées Non-Disponibles",
+            "UNDERREPL_RECORDS": "Entrées Sous-Répliquées",
+            "SIZE": "Taille"
+        },
+        "SCHEMA": "Schema",
+        "TYPE": "Type"
+    },
+    "VIEW": {
+        "TITLE": "Vues",
+        "NAME": "Nom",
+        "TYPE": "Type",
+        "DEFINITION": "Définition",
+        "COLUMNS": "Colonnes"
+    },
+    "NODE": {
+        "NOT_REACHABLE_MSG": "La grappe n'est pas joignable",
+        "NAME": "Nom",
+        "NODES": "Nœuds",
+        "OVERVIEW": "Aperçu",
+        "HEALTH": "Santé",
+        "HOSTNAME": "Nom d'hôte",
+        "CRATE_VERSION": "Version de CrateDB",
+        "SNAPSHOT": "instantané",
+        "HTTP_ENDPOINT": "Point de connection HTTP",
+        "CPU_USAGE": "Usage CPU",
+        "HEAP_USAGE": "Usage du tas",
+        "DISK_USAGE": "Usage Disque",
+        "LOAD": "Charge",
+        "MINUTES": "min",
+        "CPU_CORES": " CPU",
+        "CORES": "Cœurs",
```

Comment:
Je ne sais pas si `Cœurs` est le terme informatique le plus approprié. `noyaux` peut être ?

Reply:
Bon finalement ce serait plus juste de garder `Cœurs `

Reply:
Yes, `noyau` refers to the notion of `kernel` 😃